views/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
```

Comment:
This get's flagged as an accessibility issue. This should have a language `<html lang="en">`